packages/sw-routing/src/lib/express-route.js

```diff
@@ -66,8 +74,8 @@ class ExpressRoute extends Route {
    *        HTTP method. Defaults to `'GET'` if not specified.
    */
   constructor({path, handler, method}) {
-    if (path.substring(0, 1) !== '/') {
-      throw ErrorFactory.createError('express-route-requires-absolute-path');
+    if (!(path.startsWith('/') || path.startsWith('https://'))) {
```

Comment:
why no http:// ? Can't images use HTTP without causing the insecure content issue?

I'm also not 100% sw-helpers should be enforcing this - but instead leave the browser to enforce this.

Also is this checked needed? This was only added to block cross origin urls.